src/main/java/racingcar/RacingGameMain.java

```diff
@@ -0,0 +1,44 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class RacingGameMain {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        Random random = new Random();
+        RacingGame racingGame = new RacingGame();
+        WinnerExtraction winnerExtraction = new WinnerExtraction();
+
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        String userInput = scanner.nextLine();
+        String[] carNameArr = userInput.split(",".trim());
```

Comment:
```suggestion
        String[] carNameArr = userInput.split(",").trim();
```

[지난번 리뷰내용](https://github.com/next-step/java-racingcar/pull/2185#discussion_r631773415)

Reply:
split메서드뒤에 .trim()을 붙이니까 에러가 납니다ㅠ

Reply:
`userInput`에 `trim()`해야 하는데, 잘 못 작성했네요.